cleo_plugins/Audio/CAudioStream.cpp

```diff
@@ -221,11 +223,16 @@ void CAudioStream::Process()
         }
     }
 
+    float prevSpeed = speed.value();
+    float prevVolume = volume.value();
+
     // update animated params
     speed.update(CSoundSystem::timeStep);
     volume.update(CSoundSystem::timeStep);
 
-    if (volume.value() <= 0.0f)
+    // transitioning speed or volume to 0 pauses playback
```

Comment:
This is unexpected behavior.

So here are my expectations.

1. Volume should affect volume, not playback state
2. `set_x` should be an alias to `set_x_with_transition 0` (where `x` is either speed or volume)